tutor4points/core/templates/base.html

```diff
@@ -15,10 +15,8 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity=
     "sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 
-    <title> {% block title %}Tutor4Points{% endblock %} </title>
-<div class = "header">
-    <h1 > {% block heading %}Tutor4Points{% endblock%} </h1>
-</div>
```

Comment:
Create another base page for endpoints after the user registers/login to prevent navbar from showing in register and login page